mod.d/hungtasks.yaml

```diff
@@ -20,6 +20,9 @@ version: !!str 1.0
 title: !!str Detects hung tasks
 helptext: !!str |
   Detects hung tasks and gathers output (generally due to io device unresponsiveness)
+  For systems using journald instead of traditional syslog, the following optional flags are available.
+  Optional --since= shows entries on or newer than a specified date. Takes systemd.time format. Defaults to 48hr.
```

Comment:
Need to update the constraints. Applies to most modules updated in this commit (except softlockup).

Reply:
Done, good catch!